modules/custom_operations/user_ie_extensions/tokenizer/utils.cpp

```diff
@@ -0,0 +1,279 @@
+    auto begins = inputs[0].data<const int32_t>();
+    auto ends   = inputs[1].data<const int32_t>();
+    auto chars  = inputs[2].data<const uint8_t>();
+
+    // Set output shapes
+    outputs[0].set_shape(inputs[0].get_shape());
+    outputs[1].set_shape(inputs[1].get_shape());
+    const size_t num_elements = inputs[0].get_size();
+
+    // TODO: How to avoid copying from this temporary buffer?
+    // TODO: It can be possible to collect output symbols directly in the output tensor memory if `normalizer` has reasonable estimation for the final size.
+    std::deque<uint8_t> buffer;
+
+    // For the whole implementation below the input shapes can be ignored, we are working with the flatten representaions
+    // and only number of elements in the original tensors matter
+
+    // Get pointers in the output tensors
+    auto new_begins = outputs[0].data<int32_t>();
+    auto new_ends   = outputs[1].data<int32_t>();
+
+    for(size_t i = 0; i < num_elements; ++i) {
+        new_begins[i] = buffer.size();
+        std::string new_str = normalizer(std::string(chars + begins[i], chars + ends[i]));
+        buffer.insert(buffer.end(), new_str.begin(), new_str.end());
+        new_ends[i] = buffer.size();
+    }
+
+    // Copy collected symbols to the target output tensor
+
+    outputs[2].set_shape(Shape{buffer.size()});
+    auto new_chars  = outputs[2].data<uint8_t>();
+    std::copy(buffer.begin(), buffer.end(), new_chars);
+
+    return true;
+}
+
+std::shared_ptr<Node> string_attribute_to_constant (const ov::frontend::NodeContext& node, const std::string& name) {
+    // FIXME: using space to pad the value to work-around CPU issue with empty constants
+    auto value = node.get_attribute<std::string>(name) + " ";
+
+    // TODO: How to translate attribute `replace_global`?
+
+    #if USE_STRING_TENSORS
+    return std::make_shared<Constant>(element::string, Shape{}, &value);
+    #else
+    return std::make_shared<Constant>(element::u8, Shape{value.length()}, (const void*)value.data());
+    #endif
+}
+
+
+// Pack any container with string to ov::Tensor with element type u8
+// Requirements for BatchOfStrings: .size() with size and .begin(), .end() as iterators, elements with .begin(), .end() and .length()
+// so basically any STL container with std::string is compatible
+// Tensor destination will be reshaped according tha input data
+template <typename BatchOfStrings>
+void pack_strings (const BatchOfStrings& strings, ov::Tensor& destination) {
+    auto batch_size = strings.size();
+
+    // First run over all elements: calculate total memory required to hold all strings
+    auto symbols_size = std::accumulate(
+        strings.begin(), strings.end(), size_t(0),
+        [](size_t accum, typename BatchOfStrings::const_reference s)
+        { return accum + s.length(); });
+
+    auto total_size = 4*(1 + 1 + batch_size) + symbols_size;
+    destination.set_shape({total_size});
+
+    auto data = destination.data<uint8_t>();
```

Comment:
`int8_t` is semantically closer to `char`. You should change the type of string tensors to `int8_t`.